addons/base_import/models/base_import.py

```diff
@@ -1424,7 +1424,8 @@ def _handle_multi_mapping(self, import_fields, input_file_data):
                 for field in split_fields:
                     if field != target_field:  # if not on the last hierarchy level, retarget the model
                         target_model = self.env[target_model][field]._name
-                field_type = self.env[target_model].fields_get().get(target_field, {}).get('type', '')
+                field = self.env[target_model]._fields.get(target_field)
```

Comment:
MessageRemark! First commit: missing explanations (import time on which kind of records ? what is the number of records to import ? what are globally the installed apps) ?

Also missing an explanation of your fix (fields_get is slow -> ok, and then ?) .